tools/tools-dev/src/command/common/run.rs

```diff
@@ -6,10 +6,14 @@ pub struct RunArgs {
     #[arg(short)]
     pub verbose: bool,
 
-    /// Build artifacts in release mode, with optimizations.
-    #[arg(short, long)]
+    /// Build artifacts with the release profile. Equivalent to "--profile release".
+    #[arg(short, name = "r", conflicts_with = "profile")]
```

Comment:
Why do you think `--profile=release` isn't sufficient? This is also reflected in the code as it conflicts.

`--profile` is explicit, `--release` for usual cargo is for distinction between debug and optimized builds, in our case it's more involved.

Reply:
Same with prove command

Reply:
Was just looking to retain parity with the previous args (at least for `cargo nexus run`). Happy to remove if we want the user to be fully explicit.

Reply:
I guess I agree it should be disabled by default if we think there is a good chance the translation will not work with release. Probably we should go back to a 32-bit encoding so the we don't have this issue.